addons/udes_delivery_control/models/stock_picking.py

```diff
@@ -0,0 +1,107 @@
+from odoo import fields, models, api
+
+EXTRA_FIELDS = [
```

Comment:
Make this a set for better comparison speed :)

Reply:
done

Reply:
done